app/src/main/assets/sf_airport_route.geojson

```diff
@@ -0,0 +1,1638 @@
+      {
+  "type": "FeatureCollection",
+  "features": [
```

Comment:
nit: can we download this geojson from network, if it's taken from another js example?